framework/wazuh/core/tests/test_rootcheck.py

```diff
@@ -177,3 +177,19 @@ def test_last_scan(mock_connect, mock_send, mock_info):
 
 
 remove_db(test_data_path)
+
+
+@pytest.mark.parametrize('agent', ['001', '002', '003'])
+@patch('wazuh.core.wdb.WazuhDBConnection')
+def test_rootcheck_delete_agent(mock_db_conn, agent):
+    """ Test if proper parameters are being sent to the wazuhdb socket
+
+    Parameters
+    ----------
+    agent : str
+        Agent whose information is being deleted from the db
+    mock_db_conn : WazuhDBConnection
+        Object used to send the delete message to the wazuhdb socket
```

Comment:
Each parameter description should end with a period:

```suggestion
        Agent whose information is being deleted from the db.
    mock_db_conn : WazuhDBConnection
        Object used to send the delete message to the wazuhdb socket.
```